### ISSUE TITLE
Make it easier to find other room visitors

### ISSUE THREAD

**Issue by emclaren** [mozilla/hubs#2585]
**Is your feature request related to a problem? Please describe.**
In large rooms, its sometimes hard to know where to go to find other people in the space. 

**Describe the solution you'd like**
It would be nice if there was some indication where other people are in the world ( e.g. a map, or an indicator). Or if you could teleport to be near another person from by selecting them them from the visitor list. 

**Additional context**
I've received this feedback from community members, and I have also experienced a need for this myself in larger spaces. The A11Y community also expressed that it can sometimes be challenging to find folks in world. Here is some a description I received from a user about this: 

" Once my audio was setup, avatar picked and name entered, I was in the room. I didn't see or hear anyone. Luckily, I had joined early on my laptop and roamed around a bit before remembering my Quest was downstairs, so I knew where to go. However, as a first experience via XR, it was definitely a bit lonely feeling with no clear indicators on where to go."

**Comment by badcat** [comment]
I like the Teleport idea.  I'd guess anything similar to the object "Go to" feature would work.

**Comment by emclaren** [comment]
feedback from recent a11y user test shared with us mentioned this issue: "Opening the user list (Figure 5) was successful, and the names were screen readeraccessible. When selecting a user, though, no functionality was present other than displaying its avatar. ​A chat functionality or a way to locate or interact with the user through this list would be desirable."

**Comment by TechThomas** [comment]
Yes I agree, the "Go To" functionality would be really powerful.  I believe VirBELA has a similar feature that worked pretty well and might be good to compare against https://www.virbela.com/

**Comment by djay** [comment]
In discord with multiple voice channels its awkward when someone enters and exits a room. It would be polite to try and teleport them near the person not right into the middle of a conversation someone is having with someone else.

Perhaps an alternative would be a "walk to" action esp. if you can walk fast?
